dot/state/storage_test.go

```diff
@@ -31,6 +30,8 @@ func newTestStorageState(t *testing.T) *StorageState {
 }
 
 func TestStorage_StoreAndLoadTrie(t *testing.T) {
+	t.Skip()
```

Comment:
does this skip have a reason? can you introduce a `TODO` comment explaining why or pointing to an issue

Reply:
Oh sorry! I almost missed it while I was working, thanks for detecting it!

Reply:
Solved! thanks for the good catch!

Reply:
@dimartiro the skip should be removed or is it intentional? if so could you add a TODO here? plz 😅

Reply:
Weird, seems that I didn't push it